### ISSUE TITLE
Version on master doesn't work

### ISSUE THREAD

**Issue by grenmester** [Goles/Battery#36]
The current version of `battery` on the `master` branch is not functional. I have been trying to fix the code and I have found the offending commit to be 417a2c0.

It introduces a syntax error which I brought up in #33. It also runs into an error of not being able to correctly set variables in a pipeline in the `battery_charge` function. Information about this issue can be found [here](http://mywiki.wooledge.org/BashFAQ/024).

`battery` works fine prior to the aforementioned commit so there are two options as I see it. One is reverting back to the version before the commit and adding the features that have been added since the commit (zsh support, -e option, linux support). Another is fixing the code and resolving the issue with assigning values to variables in pipelines. I am unable to do the second option since I am not well-versed in `bash` but I should be able to revert to the old version and implement zsh support and the -e option. Thoughts?

**Comment by Goles** [comment]
Hey!

First of all, thanks for taking the time to look at this. It would be nice if you can PR a code reversal.

Thanks.

**Comment by grenmester** [comment]
So I realized that it was my bash that was acting up. However, I looked at the `zsh` implementation and it merely escapes that characters which doesn't fix the original issue I had with putting it in my right prompt. I fixed this by using proper `zsh` prompt syntax. I have made a pull request for this.

However, the default colors that I have chosen match the color scheme for solarized. If you want to change the default colors for the `zsh` option, feel free.
